### PR TITLE
po/help new module

### PR DESCRIPTION
**Pull request by TurboGit**
Help is now available for dev & standard releases.

This patch-set contains also fixes  for using the integrated help on buttons. We don't want to send the press/release action to the button being checked for help. It also adds some helps lighttable layout.

### CODE REVIEW

**Comment by elstoc** [comment]
Seems to work pretty well on dev. There are a few individual links that are incorrect, but I'll double-check those before 3.6 is released.

File: src/libs/tools/global_toolbox.c [hunk 1]
```diff
@@ -633,7 +638,10 @@ static void _main_do_event(GdkEvent *event, gpointer data)
               }
             }
             if(!is_language_supported) lang = "en";
-            char *url = g_build_path("/", base_url, lang, help_url, NULL);
+            char *url = dt_is_dev_version()
+              ? g_build_path("/", base_url, help_url, NULL)
+              : g_build_path("/", base_url, lang, help_url, NULL);
```
Comment (elstoc) [inline src/libs/tools/global_toolbox.c:643]: At some point, for the non-dev version we'd like to put the version number (X.Y) in the URL. If we can get darktable.org transferred to a new server soon enough I hope we can set this up early for testing.

File: src/common/usermanual_url.c [hunk 2]
```diff
@@ -34,14 +34,15 @@ dt_help_url urls_db[] =
   {"image",                      "module-reference/utility-modules/lighttable/selected-image/"},
   {"copy_history",               "module-reference/utility-modules/lighttable/history-stack/"},
   {"styles",                     "module-reference/utility-modules/lighttable/styles/"},
+  {"timeline",                   "module-reference/utility-modules/lighttable/timeline/"},
   {"styles_usage",               "module-reference/utility-modules/lighttable/styles/#module-controls"},
   {"metadata",                   "module-reference/utility-modules/shared/metadata-editor/"},
   {"tagging",                    "module-reference/utility-modules/shared/tagging/"},
-  {"geotagging",                 "module-reference/utility-modules/lighttable/geotagging/"},
+  {"geotagging",                 "module-reference/utility-modules/shared//geotagging/"},
```
Comment (xfh) [inline src/common/usermanual_url.c:41]: Looks like one slash too much?

Comment (TurboGit) [reply src/common/usermanual_url.c]: Good catch ! Thanks.

File: src/libs/tools/global_toolbox.c (hunk 1 above)
Comment (TurboGit) [reply src/libs/tools/global_toolbox.c:647]: Ok, I'll wait for this before merging then.

**Comment by elstoc** [comment]
@TurboGit the color balance rgb link should work now - I've pushed a dummy page to dtdocs pending the completed documentation.

**Comment by TurboGit** [comment]
@elstoc : so should I merge this now? and work on the versioning of the doc in another PR?

**Comment by elstoc** [comment]
Yeah I think so - it delivers up-to-date help links in master, which is a nice novelty.

**Comment by TurboGit** [comment]
Ok merged then! Thanks for review.

**Pull request merged**